package_manifest.yaml

```
@@ -438,6 +461,9 @@ yggdrasil_client_packages:
 
 foreman_client_packages:
   vars:
+    build_package_build_system: copr
+    copr_project_user: 'theforeman'
+    copr_project_name: 'foreman-client-nightly'
```

I guess this should be 
```suggestion
    copr_project_name: 'foreman-client-{{ foreman_version }}'
```